ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -53,6 +53,8 @@
   public static final String KSQL_MIGRATIONS_TOPIC_REPLICAS = "ksql.migrations.topic.replicas";
   public static final int KSQL_MIGRATIONS_TOPIC_REPLICAS_DEFAULT = 1;
 
+  public static final String KSQL_MIGRATIONS_DIRECTORY_PATH = "ksql.migrations.dir.override";
```

Comment:
nit: It looks like the variable name and the string are precisely just uppercase->lowercase and '_' replaced with '.'s.